app/api/client_resources.py

```diff
@@ -290,7 +290,7 @@ def resource_update(
         is_browser_shortcut_enabled: bool,
         remote_network_id: str,
         security_policy_id: str | None,
-        protocols: ResourceProtocols,
```

Comment:
why?

Reply:
`protocols` is passed in as a dictionary, not a Pydantic model. It's converted at the call-site in [to_graphql_argument()](https://github.com/Twingate/kubernetes-operator/pull/613/files#diff-3741cb2a9ecf922677b6680e54a6df3ae14d768feace19394a0729053498829fR149). We update the [`resource_method`](https://github.com/Twingate/kubernetes-operator/blob/a623091f3724698d9f6193a38342fac89f5690c9/app/api/client_resources.py#L262) but forgot to update this method's signature.